src/detect-sip-protocol.c

```diff
@@ -52,24 +52,23 @@
 #include "stream-tcp.h"
 
 #include "rust.h"
-#include "app-layer-sip.h"
 
 #define KEYWORD_NAME "sip.protocol"
 #define KEYWORD_DOC  "sip-keywords.html#sip-protocol"
 #define BUFFER_NAME  "sip.protocol"
 #define BUFFER_DESC  "sip protocol"
-static int g_buffer_id = 0;
+ static int g_buffer_id = 0;
```

Comment:
clang-format did this

Reply:
Why is clang misbehaving like that? Wonder if it could be an interference from the `#define` line above this one... 🤔

Reply:
I found this: https://zed0.co.uk/clang-format-configurator/

But scanning over the documentation and testing with that I couldn't find a possible option to affect that...

Reply:
> Why is clang misbehaving like that?

No idea. Is this worth investigating ?

Reply:
Also don't know, but seems a pretty odd formatting choice, and doesn't seem to agree to our standards, just like that other case with the switch-case statements...

Reply:
`diff -y src/detect-sip-protocol.c src/detect-sip-method.c` showed me the answer

src/detect-sip-protocol.c starts with a leading space before the comment copyright license

Reply:
😯